.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "semi": false,
+    "trailingComma": "all",
+    "singleQuote": true
```

Comment:
should this have similar settings to editorconfig?

Reply:
Possible, it seems to work though, I wonder if it picks it up. You mean things like tab width?